go/vt/vttablet/faketmclient/fake_client.go

```diff
@@ -36,7 +36,9 @@ import (
 	logutilpb "vitess.io/vitess/go/vt/proto/logutil"
 	querypb "vitess.io/vitess/go/vt/proto/query"
 	replicationdatapb "vitess.io/vitess/go/vt/proto/replicationdata"
+	"vitess.io/vitess/go/vt/proto/tabletmanagerdata"
```

Comment:
Also, nit on things like this: you've now imported the package twice 😄, vitess codebase prefers to rename any protobuf package (with a few exceptions, like `vttime` and `vtrpc`), as `${pkgname}pb`